qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -3,7 +3,10 @@
     no Fedora.8, Fedora.9, Fedora.10, Fedora.11, Fedora.12, Fedora.13, Fedora.14, Fedora.15
     type = qemu_guest_agent
     gagent_name = "org.qemu.guest_agent.0"
-    gagent_install_cmd = "rpm -q qemu-guest-agent || yum install -y qemu-guest-agent"
+    gagent_install_cmd = "yum install -y qemu-guest-agent"
+    gagent_start_cmd = "service qemu-guest-agent start"
+    gagent_status_cmd = "service qemu-guest-agent status"
+    gagent_pkg_check_cmd = "rpm -qa |grep qemu-guest-agent"
```

Comment:
Can simply check this by command `rpm -q qemu-guest-agent`.

Reply:
Thanks, modified.